openrtb_ext/response.go

```diff
@@ -41,6 +41,7 @@ type ExtResponseSyncData struct {
 type ExtResponsePrebid struct {
 	AuctionTimestamp int64           `json:"auctiontimestamp,omitempty"`
 	Passthrough      json.RawMessage `json:"passthrough,omitempty"`
+	Modules          json.RawMessage `json:"modules,omitempty"`
```

Comment:
Actually, at the moment I cannot remember any specific reason for choosing the `json.RawMessage` over a nested structure.
Most likely because the module data is only meant to be sent with a response so there is no need to maintain one more nested struct. Also, the value for the `Modules` field is set only after the `BidResponse` has already been created, thus we do not have access to the parsed `ExtBidResponse` structure and we deal with the `ext` field in JSON format: https://github.com/prebid/prebid-server/blob/9c9a40cf70da1375ab4feb4b475191bd52060005/endpoints/openrtb2/auction.go#L286